src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlAudienceRestrictionCondition.cs

```diff
@@ -34,6 +34,20 @@ public SamlAudienceRestrictionCondition(IEnumerable<Uri> audiences)
             Audiences = (audiences == null) ? throw LogArgumentNullException(nameof(audiences)) : new List<Uri>(audiences);
         }
 
+        /// <summary>
+        /// Creates an instance of <see cref="SamlAudienceRestrictionCondition"/>.
+        /// </summary>
+        /// <param name="audiences">An <see cref="IEnumerable{String}"/> containing the audiences for a <see cref="SamlAssertion"/>.</param>
+        internal SamlAudienceRestrictionCondition(IEnumerable<string> audiences)
```

Comment:
Why is this method internal?

Reply:
It was written before I was told not to use internals and I missed it :)... Should we make it public or remove it and require the cast to Uri to happen outside of this class?